src/Stripe/Infrastructure/Mapper.cs

```diff
@@ -14,8 +15,9 @@ public static List<T> MapCollectionFromJson(string json, string token = "data")
 
             var allTokens = jObject.SelectToken(token);
             foreach (var tkn in allTokens)
-                list.Add(Mapper<T>.MapFromJson(tkn.ToString()));
-
+            {
+                list.Add(MapFromJson(tkn.ToString()));
+            }
```

Comment:
I know you don't, and I tried not to make any changes. I added them here because this is where the ICloneable problem sprung up, and I was trying a few different things to fix it. In theory, it's fine to keep your code cleaner, but in practice it's the same issue that caused that OpenSSL bug last year. You're welcome to take them out after the checkin, but if you don't mind me being honest, it's not the best coding habit. ;)